### PR TITLE
[xdoctest] reformat example code with google style in No.91-94

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Others
### Description

- ./python/paddle/nn/layer/loss.py
- ./python/paddle/nn/decode.py
- ./python/paddle/nn/quant/format.py

预览：

en: http://preview-paddle-pr-55832.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/abs_en.html
zh: http://preview-paddle-pr-55832.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/abs_cn.html

存在的问题：
./python/paddle/nn/clip.py 中，代码和示例存在不少问题，还是fluid的，基于1.8的代码。需要看看如何修改。

关联 PR：
https://github.com/PaddlePaddle/Paddle/issues/55629
https://github.com/PaddlePaddle/Paddle/pull/55295

@sunzhongkai588 @SigureMo @megemini

### CODE REVIEW

File: python/paddle/nn/quant/format.py [hunk 1]
```diff
@@ -149,29 +149,28 @@ class ConvertibleQuantedLayer(Layer, metaclass=abc.ABCMeta):
...
+            ...         self.weight_b = paddle.create_parameter(shape=[1], dtype='float32')
+            ...         self.quanter_for_weight_a = None
+            ...         self.activation_weight = None
+            ...     def forward(self, input):
+            ...         qweight_a = self.quanter_for_weight_a(self.weight_a)
+            ...         weight_b = self.weight_b
+            ...         qinput = self.activation_weight(input)
+            ...         # compute with qweight_a, weight_b and qinput.
+            ...         return qweight * qinput + weight_b
+            ...      def weights_to_quanters(self):
```
Comment (SigureMo) [inline python/paddle/nn/quant/format.py]: 这里缩进怎么没对齐？

File: python/paddle/nn/layer/loss.py [hunk 2]
```diff
@@ -1820,12 +1849,14 @@ class TripletMarginLoss(Layer):
...
-            # Tensor([0.        , 0.57496738, 0.        ])
+            Tensor(shape=[3], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [0.        , 0.57496595, 0.        ])
 
             triplet_margin_loss = paddle.nn.TripletMarginLoss(margin=1.0, swap=True, reduction='mean', )
             loss = triplet_margin_loss(input, positive, negative,)
             print(loss)
-            # Tensor(0.19165580)
+            Tensor(shape=[], dtype=float32, place=Place(cpu), stop_gradient=True,
+            2.40039468)
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py:1846]: 这段代码怎么没改格式？

File: python/paddle/nn/layer/loss.py [hunk 3]
```diff
@@ -642,29 +647,29 @@ class L1Loss(Layer):
...
-            print(output)
-            # Tensor(shape=[2, 2], dtype=float32, place=Place(gpu:0), stop_gradient=True,
-            #        [[0.20000005, 0.19999999],
-            #         [0.20000000, 0.79999995]])
+            >>> l1_loss = paddle.nn.L1Loss(reduction='none')
+            >>> output = l1_loss(input, label)
+            >>> print(output)
+            Tensor(shape=[2, 2], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[0.20000005, 0.19999999],
+            [0.20000000, 0.79999995]])
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]:
```suggestion
            [[0.20000005, 0.19999999],
             [0.20000000, 0.79999995]])
```
对齐一下吧

File: python/paddle/nn/layer/loss.py [hunk 4]
```diff
@@ -439,21 +439,25 @@ class HSigmoidLoss(Layer):
...
+            [ 0.46585739,  0.95573163, -0.91368192]])
+            >>> label = paddle.to_tensor([0, 1, 4, 5])
+            >>> m = paddle.nn.HSigmoidLoss(3, 5)
+            >>> out = m(input, label)
+            >>> print(out)
+            Tensor(shape=[4, 1], dtype=float32, place=Place(cpu), stop_gradient=False,
+            [[2.22472525],
+            [1.97849643],
+            [1.69004655],
+            [2.50623918]])
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]:
```suggestion
            [[2.22472525],
             [1.97849643],
             [1.69004655],
             [2.50623918]])
```

File: python/paddle/nn/layer/loss.py [hunk 5]
```diff
@@ -439,21 +439,25 @@ class HSigmoidLoss(Layer):
...
+            >>> import paddle
+            >>> paddle.set_device('cpu')
+            >>> paddle.seed(2023)
+            >>> input = paddle.uniform([4, 3])
+            >>> print(input)
+            Tensor(shape=[4, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[ 0.73167229,  0.04029441, -0.48078126],
+            [ 0.81050646, -0.15199822, -0.18717426],
+            [ 0.94041789,  0.48874724,  0.03570259],
+            [ 0.46585739,  0.95573163, -0.91368192]])
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]:
```suggestion
            [[ 0.73167229,  0.04029441, -0.48078126],
             [ 0.81050646, -0.15199822, -0.18717426],
             [ 0.94041789,  0.48874724,  0.03570259],
             [ 0.46585739,  0.95573163, -0.91368192]])
```

File: python/paddle/nn/layer/loss.py [hunk 6]
```diff
@@ -439,21 +436,25 @@ class HSigmoidLoss(Layer):
...
+            >>> paddle.seed(2023)
+            >>> input = paddle.uniform([4, 3])
+            >>> print(input)
+            Tensor(shape=[4, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[ 0.73167229,  0.04029441, -0.48078126],
+             [ 0.81050646, -0.15199822, -0.18717426],
+             [ 0.94041789,  0.48874724,  0.03570259],
+             [ 0.46585739,  0.95573163, -0.91368192]])
+            >>> label = paddle.to_tensor([0, 1, 4, 5])
+            >>> m = paddle.nn.HSigmoidLoss(3, 5)
```
Comment (SigureMo) [inline python/paddle/nn/layer/loss.py]: image

最后一个数据怎么差异这么大呢？而且只有这一个有差异

Comment (sunzhongkai588) [reply python/paddle/nn/layer/loss.py]:
```suggestion
            >>> m = paddle.nn.HSigmoidLoss(3, 6)
```
看了下，`num_classes`的参数输入改成`6`试试（如上），因为 label = paddle.to_tensor([0, 1, 4, 5]) 好像至少有6个class（0到5） @enkilee

Comment (enkilee) [reply python/paddle/nn/layer/loss.py]: 孙师傅太厉害啦！

**Review by SigureMo — APPROVED** [review]
LGTMeow~
